src/Hubbup.Web/Client/Standup/components.tsx

```diff
@@ -0,0 +1,370 @@
+            let title = issue.milestone ? issue.milestone.title : "< No Milestone >";
+            milestoneBadge = <Badge title={title} />;
+        }
+
+        return <li className="list-group-item">
+            <div className="row">
+                <IssueUserView issue={issue} />
+                <div className="col-md-9">
+                    <div>
+                        <span className="repo-name">
+                            {issue.repository.owner.login}/{issue.repository.name}
+                        </span>
+                        <strong>{issue.title}</strong>
+                    </div>
+
+                    {issue.labels.map(label => <LabelView key={label.id} label={label} />)}
+
+                    {issue.working ?
+                        <AgeBadge date={issue.workingStartedAt} timeAgo={issue.workingTimeAgo} prefix="Started working" stale={issue.stale} /> :
+                        <AgeBadge date={issue.createdAt} timeAgo={issue.createdTimeAgo} prefix="Opened" stale={issue.stale} />}
+
+                    {milestoneBadge}
+                </div>
+            </div>
+        </li>;
+    }
+}
+
+class IssueList extends React.Component<{ id: string, issues: Data.Issue[], emptyText: string, title: string, collapsable?: boolean }, undefined> {
+    render() {
+        let content;
+        let collapser;
+        if (this.props.issues.length > 0) {
+            content = <ul className="list-group">
+                {this.props.issues.map(issue => <IssueView key={issue.id} issue={issue} />)}
+            </ul>;
+            if (this.props.collapsable) {
+                collapser = <button className="btn btn-xs btn-info chevronButton" type="button" data-toggle="collapse" data-target={`#collapse-otherissues-${this.props.id}`}>
+                    <span id="chevronGlyph" className="glyphicon glyphicon-chevron-right"></span>
+                    {' ' /* https://facebook.github.io/react/blog/2014/02/20/react-v0.9.html#jsx-whitespace */}
+                    Show/hide
+                </button>;
+                content = <div className="collapse" id={`collapse-otherissues-${this.props.id}`} style={{ "marginTop": "10px" }}>
+                    {content}
+                </div>;
+            }
+        }
+        else {
+            content = <div className="alert alert-info">{this.props.emptyText}</div>;
+        }
+
+        return <div>
+            <h4>
+                {this.props.title}
+                {' ' /* https://facebook.github.io/react/blog/2014/02/20/react-v0.9.html#jsx-whitespace */}
+                <span className="badge">{this.props.issues.length}</span>
+                {' ' /* https://facebook.github.io/react/blog/2014/02/20/react-v0.9.html#jsx-whitespace */}
+                {collapser}
+            </h4>
+            {content}
+        </div>;
+    }
+}
+
+type RateLimitUpdater = (graphQl: { cost: number, remaining: number, resetAt: string }, rest: { remaining: number, reset: string }) => void;
```

Comment:
Actually this needn't be a `type` or `interface` at all. I added it when I thought I was going to use it in multiple places but I didn't end up doing so.